src/ssh/azext_ssh/custom.py

```diff
@@ -49,10 +51,17 @@ def ssh_vm(cmd, resource_group_name=None, vm_name=None, ssh_ip=None, public_key_
     credentials_folder = None
 
     op_call = ssh_utils.start_ssh_connection
+    if winrdp:
+        if platform.system() != 'Windows':
```

Comment:
I want to confirm with you that the judgment here does not need to ignore the case of `platform.system()`?